### ISSUE TITLE
Design a Mockup for FlashProcessing UI

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#28]
## User story
1. As a user
2. I want to see a visual representation of the FlashProcessing UI
3. So that understand how to initiate and verify the flashing process in the GUI

## Acceptance criteria
* Visual representations of flashing process initiation and verification elements are included
* Buttons, progress indicators, and feedback messages are also shown

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
